server/src/modules/users/users.service.ts

```diff
@@ -28,6 +28,8 @@ import { MapFavoriteDto } from '@common/dto/map/map-favorite.dto';
 import { ConfigService } from '@nestjs/config';
 import { UsersGetAllQuery } from '@common/dto/query/user-queries.dto';
 import { SteamUserSummaryData } from '@modules/auth/auth.interfaces';
+import { MapDto } from '@/common/dto/map/map.dto';
+import { MapSummaryDto } from '@/common/dto/user/user-maps-summary.dto';
```

Comment:
Should be `@common` not `'@/common`. This works because `@/` is just `src/` but the idea with these named import paths is to avoid that hierarchy.